src/scss/components/nav.scss

```diff
@@ -92,29 +94,38 @@
   }
 
   // Mobile only
-  @include media-breakpoint-down(md) {
+  @include media-breakpoint-down(sm) {
     .navbar-nav {
-      margin-top: 0;
-      margin-bottom: 0;
+      margin: 0;
     }
 
     .navbar-collapse {
       box-shadow: none;
-    }
-
-    .navbar-form {
-      @include theme-color("header", "background-color");
-      margin-top: 0;
       border-top: 1px solid $navbar-dark-color;
       border-color: $navbar-dark-color;
+      margin-left: -20px;
+      margin-right: -20px;
```

Comment:
Convert this to rems please

Reply:
I think in this case it needs to be px since it is compensating for the bootstrap column padding which is defined in px. However I this is related to $grid-gutter-width bootstrap variable and so could change this to -($grid-gutter-width / 2)